openshift/templates/jenkins-ephemeral-monitored.json

```diff
@@ -0,0 +1,339 @@
+{
+  "kind": "Template",
```

Comment:
We need to put these templates in openshift/origin .
@gabemontero does origin still accept new templates?

Reply:
it does, but as we commented in the openshift/origin PR, you add the template here in this repo, and then update https://github.com/openshift/library 's jenkins artifacts to reference your new template here once this PR merges.

Eventually, as part of 4.3, I'll pull in your new template into the samples operator.

the template refs in openshift/origin are pretty much historical at this point ... all the "samples" repos and the openshift/library content are the "source of truth"

Reply:
ok @gabemontero , we will do that then.